packages/zoe/src/zoeService/utils.d.ts

```diff
@@ -101,6 +101,7 @@ export type StartInstance = <SF>(
   // 'brands' and 'issuers' need not be passed in; Zoe provides them as StandardTerms
   terms?: Omit<StartParams<SF>['terms'], 'brands' | 'issuers'>,
   privateArgs?: StartParams<SF>['privateArgs'],
+  label?: string,
```

Comment:
more than once I added a label without filling in `undefined` for `privateArgs` and it cost a full blockchain start to discover the problem. The runtime interface guard requires privateArgs to be a record; it would be nice if we could get an early static error for putting a string where the privateArgs go.

Reply:
I'll take a look today. I think we can land this as is, if you want the labels sooner